ymca/src/main/webapp/js/LoggingEndpoint.js

```diff
@@ -14,11 +14,21 @@ var loggingEndpoint = {
 			
 			websocket.onmessage = function(message) 
 			{
-				var jsonMessage = JSON.parse(message.data);
-				console.log(jsonMessage['level'] + ": " + jsonMessage['message']);
+				var jsonMessage = JSON.parse(message.data);				
+				loggingEndpoint.log(jsonMessage.level, jsonMessage.message);
 			};
 		},
 		
+		log(level, message)
+		{
+			if(typeof outputLog === "object")
+			{
+				outputLog.append(level, message);
+			}
+			
+			console.log(level + ": " + message);
```

Comment:
would love if it could use `console.log`, `console.info`, `console.warn` and `console.error` etc depending on level.